core/src/main/resources/openapi/web3signer-eth2.yaml

```diff
@@ -169,20 +169,19 @@ components:
     DepositSigning:
       type: object
       properties:
+        type:
+          type: "string"
         deposit:
           type: object
           properties:
-            type:
-              type: "string"
-            signingRoot:
```

Comment:
We were keeping signingRoot as a field for now on all signing request types for compatibility with the spec so this field is still allowed. And whilst we don't use the value directly we compute the signing root from the included data and if the signingRoot field is present we will validate that is matches computed signingRoot value.

Reply:
Indeed, I misread the implementation! Thanks for the clarification, I replaced that with a description reflecting your comment